src/components/Sweep.tsx

```diff
@@ -41,19 +41,21 @@ export const Sweep = ({
   } = useWallet()
   const translate = useTranslate()
   const asset = useAppSelector(state => selectAssetById(state, assetId))
+  const feeAsset = useAppSelector(state => selectFeeAssetById(state, assetId))
 
   const {
     data: estimatedFeesData,
     isLoading: isEstimatedFeesDataLoading,
     isSuccess: isEstimatedFeesDataSuccess,
   } = useGetEstimatedFeesQuery({
     amountCryptoPrecision: '0',
+    feeAssetId: feeAsset?.assetId ?? '',
```

Comment:
Empty string opt-chain is ugly but consistent here

Reply:
since sweep is utxo only we could just use `assetId` since it is always native asset, this works and is consistent though as you stated

Reply:
cdd9810618